lighthouse-core/test/audits/dobetterweb/no-vulnerable-libraries-test.js

```diff
@@ -43,6 +43,44 @@ describe('Avoids front-end JavaScript libraries with known vulnerabilities', ()
     assert.equal(auditResult.details.items[0].detectedLib.url, 'https://snyk.io/vuln/npm:angular?lh=1.1.4&utm_source=lighthouse&utm_medium=ref&utm_campaign=audit');
   });
 
+  describe('handles vuln details correctly', () => {
+    // Vulnerability with an array of semver ranges
+    const mockSnykDb = {
+      npm: {
+        badlib: [
+          {id: 'badlibvuln:12345', severity: 'medium', semver: {vulnerable: ['<2', '>=3.0.0']}},
+        ],
+      },
+    };
+    beforeEach(() => {
+      origSnykDb = NoVulnerableLibrariesAudit.snykDB;
+      NoVulnerableLibrariesAudit.setSnykDBForTest(mockSnykDb);
+    });
+    afterEach(() => {
+      NoVulnerableLibrariesAudit.setSnykDBForTest(undefined);
+    });
+
+    it('fails when libraries w/ vulnerabilities are detected (in the semver range array)', () => {
```

Comment:
super nit: `later in the semver array` or `anywhere in the semver array` just to make it super clear since the mock is shared